.github/workflows/ci.yml

```diff
@@ -85,6 +85,41 @@ jobs:
         chmod +x codecov
         ./codecov
 
+  #
+  # Android build job
+  #
+  android:
+    runs-on: macos-latest
+    if: "! contains(toJSON(github.event.commits.*.message), '[skip ci]')"
+
+    # Definition of the build matrix
+    strategy:
+      matrix:
+        java: [ 11 ]
+        entry:
+          - { mock-maker: 'mock-maker-default', member-accessor: 'member-accessor-default' }
+
+    # All build steps
+    steps:
+    - name: 1. Check out code
+      uses: actions/checkout@v3 # https://github.com/actions/checkout
```

Comment:
Unnecessary comment is unnecessary ;)
All actions are GitHub org/repo@tag (or hash) format. So this mapping is the default resolution.